floodviz/views.py

```diff
@@ -1,7 +1,44 @@
-from flask import request, render_template, abort
+import json
+import requests
+from flask import render_template
 
 from . import app
+from . import hydrograph_utils
+
 
 @app.route('/home/')
 def home():
-    return render_template('index.html')
\ No newline at end of file
+    # new data store #
+    sites = app.config['SITE_IDS']
+    start_date = app.config['START_DT']
+    end_date = app.config['END_DT']
+    n_show_series = app.config['N_SERIES']
+    hydro_meta = app.config['HYDRO_META']
+    url_top = app.config['NWIS_SITE_SERVICE_ENDPOINT']
+
+    # Set up to retrieve all site ids from url #
+    site_string = ''
+    for idx, site in enumerate(sites):
```

Comment:
Take a look at the string join method. It will reduce this for loop to a single statement.